packages/diagram_capture/lib/diagram_capture.dart

```diff
@@ -206,7 +206,12 @@ class _DiagramFlutterBinding extends BindingBase
 
 /// A generator used by [DiagramController] for generating filenames for frames
 /// in animated diagram sequences.
-typedef File AnimationFilenameGenerator();
+typedef AnimationFilenameGenerator = File Function();
+
+/// A keyframe function designed to be executed before a particular frame is
+/// captured. Passes the current value of "now" (the time elapsed since the
+/// beginning of the animation).
+typedef DiagramKeyframe = void Function(Duration duration);
```

Comment:
i think we're still using the old style, i don't think we've enabled the new lint yet